test/s3_options.js

```diff
@@ -30,5 +30,11 @@ export default {
   cloudfrontInvalidateOptions: {
     DistributionId: CLOUDFRONT_DISTRIBUTION_ID,
     Items: ['/*']
-  }
+  },
+
+  indexOptions: {
```

Comment:
The indexOptions contain the IndexDocument, which is the name of your index.html file. e.g.: index.d9w392.html
Cloudfront: true means that the rootdocument of the cloudfront distribution will be updated to that indexdocument.
S3: true means that the index.html of the s3 website will be updated to that indexdocument.